Vizzuality/landgriffon
# Add edit modal and delete user

### General description


This adds the 'edit' button to the user list, with edit the form disabled and the 'delete user' functionality working
It changes the users list to don't display the current user name

It updates the users page style and table content 


### Designs

[users list](https://www.figma.com/file/7bmF29CIXDJwCpuloW5Nt6/%F0%9F%94%B5-Landgriffon---Redesign_BLUE?type=design&node-id=1870-167404&mode=dev)
[edit user modal](https://www.figma.com/file/7bmF29CIXDJwCpuloW5Nt6/%F0%9F%94%B5-Landgriffon---Redesign_BLUE?type=design&node-id=1870-169568&mode=dev)

### Testing instructions

1. With an admin account, go to '/profile/users'
2. the list should NOT display your current user name
3. click on some 'edit' button on the list. A modal should open with the edit user form and the 'save' button should be disabled
4. At the bottom of the list are some test users to be deleted. Try to delete them.

### Tasks
https://vizzuality.atlassian.net/browse/LANDGRIF-1346
https://vizzuality.atlassian.net/browse/LANDGRIF-1386
